### PR TITLE
bat, gdb: add Indonesian translation

### PR DESCRIPTION
**Pull request by amarps**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/bat.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# cat
```
Comment (navarroaxel) [inline pages.id/common/bat.md:1]:
```suggestion
# bat
```

File: pages.id/common/bat.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+> Mencetak dan menggabungkan berkas.
+> klon dari `cat` dengan sintaks berwarna dan integrasi Git.
+> Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/cat>.
```
Comment (navarroaxel) [inline pages.id/common/bat.md:5]:
```suggestion
> Informasi lebih lanjut: <https://github.com/sharkdp/bat>.
```

File: pages.id/common/gdb.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# gdb
+
+> GNU Debugger.
+> Informasi lebih lanjut: <https://www.gnu.org/software/gdb>.
+
+- Menjalankan debug pada sebuah berkas yang dapat dieksekusi:
+
+`gdb {{berkas-exe}}`
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:8]: We use `snake_case` for placeholders
```suggestion
`gdb {{berkas_exe}}`
```

File: pages.id/common/gdb.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- menambahkan sebuah proses pada gdb:
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:10]:
```suggestion
- Menambahkan sebuah proses pada gdb:
```

File: pages.id/common/gdb.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`gdb -p {{berkas-exe}}`
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:12]:
```suggestion
`gdb -p {{berkas_exe}}`
```

File: pages.id/common/gdb.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Menjalankan debug dengan berkas core:
+
+`gdb -c {{core}} {{berkas-exe}}`
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:16]:
```suggestion
`gdb -c {{core}} {{berkas_exe}}`
```

File: pages.id/common/gdb.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Mengeksekusi perintah GDB pada saat dijanlakan:
+
+`gdb -ex "{{perintah}}" {{berkas-exe}}`
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:20]:
```suggestion
`gdb -ex "{{perintah}}" {{berkas_exe}}`
```

File: pages.id/common/gdb.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Menjalankan gdb dan melemparkan argumen pada berkas yang dieksekusi:
+
+`gdb --args {{berkas-exe}} {{argumen1}} {{argumen2}}`
```
Comment (navarroaxel) [inline pages.id/common/gdb.md:24]:
```suggestion
`gdb --args {{berkas_exe}} {{argumen1}} {{argumen2}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**